app/components/welcome/welcome.js

```diff
@@ -156,20 +163,75 @@ const dialogs = [
       </>
     )
   },
+  ({ page, next, previous }) => {
+    return (
+      <>
+        <Back page={page} onClick={previous} />
+        <Illustration>
+          <IllustrationVault />
+        </Illustration>
+        <Heading>Introducing the Vault</Heading>
+        <p>
+          We use peer-to-peer technology: Hypergraph is powered by the people using it!
+          Share your content as you'd like, without anybody getting in between.
+        </p>
+        <p>
+          Hypergraph Vault makes your work available even when your computer is offline. Plus, we're
+          working with libraries on archiving it for future generations! 👵👨👶
+        </p>
+        <p>
+          Until January 1st, 2021, Hypergraph Vault is free, while we figure out the costs.
+          We promise to keep it to one-time fees for lifetime service.
```

Comment:
```suggestion
          It won't be more than five coffees ☕ ☕ ☕ ☕ ☕ 
```

On second thought, lets scrap the promise so we don't end up reneging. Here's something more vague but also playful, per our style :)

Reply:
To me this still feels like a promise that we might not be able to make. Also, let's say we charge €5, I'm guessing there are plenty of countries where coffee still costs < €1.

How about, "We intend to offer small one-time fees for lifetime service."

Reply:
Also difficult as we're still developing the business model. Maybe we should just delete the second sentence altogether then? Seems like a risky move then.

Reply:
Removed!